server/container_status.go

```diff
@@ -69,3 +78,30 @@ func (s *Server) ContainerStatus(ctx context.Context, req *pb.ContainerStatusReq
 	logrus.Debugf("ContainerStatusResponse: %+v", resp)
 	return resp, nil
 }
+
+func (s *Server) getMounts(id string) ([]*pb.Mount, error) {
+	config, err := s.store.GetFromContainerDirectory(id, "config.json")
+	if err != nil {
+		return nil, err
+	}
+	var m rspec.Spec
+	if err = json.Unmarshal(config, &m); err != nil {
```

Comment:
I checked the kube code and for docker it is using InspectContainer that only returns bind mounts AFAICT. 
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/docker_container.go#L332
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/docker_container.go#L351